algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/workload.py

```diff
@@ -114,24 +114,24 @@ def _build_input_queue(self,
                          global_batch_size: int,
                          num_batches: Optional[int] = None,
                          repeat_final_dataset: bool = False):
-    del data_rng
     del num_batches
     del repeat_final_dataset
     train = False
-
+    eval_train = False
     if split == 'train':
       split = 'train-clean-100+train-clean-360+train-other-500'
       train = True
     elif split == 'eval_train':
       split = 'train-clean-100+train-clean-360+train-other-500'
+      eval_train = True
     elif split == 'validation':
       split = 'dev-clean+dev-other'
     elif split == 'test':
       split = 'test-clean'
 
     ds = LibriSpeechDataset(split=split, data_dir=data_dir)
-    if split == 'eval_train':
-      indices = list(range(self.num_train_examples))
```

Comment:
should we instead just set `num_train_examples` to 263838?

Reply:
will `len(ds)` be expensive? if not, then we can leave it as is, but otherwise we should hardcode it.